docs/commands/super.md

```diff
@@ -733,19 +733,18 @@ file:
 ```
 super gharchive_gz/*.json.gz > gha.bsup
 ```
-This code path in `super` is not multi-threaded so not particularly performant but,
-on our test machine, this runs more than 2x faster than the `duckdb` method of
-creating a schema-fused table and just a bit faster than `clickhouse` could
-load the data to its beta JSON type.
+This code path in `super` is not multi-threaded so not particularly performant,
+but on our test machine it runs a bit faster than both the `duckdb` method of
+creating a schema-fused table or loading the data to the `clickhouse` beta JSON type.
```

Comment:
Even though I used the same DuckDB release version and AWS instance type as the last time I ran these tests, for some reason DuckDB did happen to run this "table creation" step substantially faster than last time (328 seconds now vs. 513 seconds last time). I can't explain it offhand and I'm not inspired to take a detour and study it deeply, but in the interest of science I've corrected the summary here to reflect that. I'll keep an eye on it in future runs and maybe the operation just turns out to have high variance.